style.css

```diff
@@ -0,0 +1,274 @@
+/*Styling of the nav bar*/
+
+nav div {
+    float: left;
+}
+
+.mid-nav-bar {
+    text-align: center;
+    display: inline-block;
+    position: relative;
+    left: 300px;
+    top: 30px;
+}
+
+input {
+    width: 500px;
+    height: 18px;
+}
+
+
+.right-nav-bar {
+    float: right;
+}
+
+.icon {
+    margin-right: 50px;
+    position: relative;
+    top: 17px;
+}
```

Comment:
Margin-right: 50 px is too much, this the cause of the overlapping
![Screenshot from 2020-07-10 12-34-11](https://user-images.githubusercontent.com/26064042/87151103-2722a700-c2ab-11ea-97f0-1c677110ecb2.png)